spec/features/organizations_list_spec.rb

```diff
@@ -24,8 +24,6 @@
         expect(subject).to have_selector('td',
           text: organization.owners.map(&:email).join(', '))
         expect(subject).to have_link('Show')
-        expect(subject).to have_link('Edit')
```

Comment:
@Gataniel why did you remove this tests? There is nothing in this task that should change it.